sym/d2c0e7b4-b53c-4325-8c61-eff6d7faf7c3/symbol.lp

```diff
@@ -0,0 +1,40 @@
+(librepcb_symbol d2c0e7b4-b53c-4325-8c61-eff6d7faf7c3
+ (name "DAC 5-Pin Single-Ended")
+ (description "A generic single-ended single-channel digital-to-analog converter with 5 pins.")
+ (keywords "dac,5-pin,single-ended,single-channel")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2018-12-02T13:46:23Z)
+ (deprecated false)
+ (category f87a2a14-4dd2-49f0-b93d-0c78ac50cf53)
+ (pin d3e9e9a3-8fbb-4a06-95e2-6d8b5bf8e380 (name "V+")
+  (position 0.0 7.62) (rotation 270.0) (length 2.54)
+ )
+ (pin 89101278-c2bf-4402-a9b0-1d3de5121086 (name "V-")
+  (position 0.0 -7.62) (rotation 90.0) (length 2.54)
+ )
+ (pin 778d00d2-6598-412e-b422-6608869e8bd1 (name "AOUT")
+  (position 12.7 0.0) (rotation 180.0) (length 2.54)
+ )
+ (pin b65ee8bf-b441-496d-9456-5d5fdeb53dfa (name "D1")
+  (position -12.7 2.54) (rotation 0.0) (length 2.54)
+ )
+ (pin 3ba7417a-5f15-44c4-947a-dd4fc3aabcd1 (name "D2")
+  (position -12.7 -2.54) (rotation 0.0) (length 2.54)
+ )
+ (polygon 3875c6eb-e784-46bd-9500-685d3c31599a (layer sym_outlines)
+  (width 0.254) (fill false) (grab_area true)
+  (vertex (position 10.16 0.0) (angle 0.0))
+  (vertex (position 2.54 5.08) (angle 0.0))
+  (vertex (position -10.16 5.08) (angle 0.0))
+  (vertex (position -10.16 -5.08) (angle 0.0))
+  (vertex (position 2.54 -5.08) (angle 0.0))
+  (vertex (position 10.16 0.0) (angle 0.0))
+ )
+ (text bc465e7b-a865-470b-9a23-4febc3886c5b (layer sym_names) (value "{{NAME}}")
+  (align right bottom) (height 3.0) (position -2.54 5.08) (rotation 0.0)
```

Comment:
Left-aligned looks much better (component names are usually very short, so it's unlikely that it ever overlaps with the V+ pin):
```suggestion
  (align left bottom) (height 3.0) (position -10.16 5.08) (rotation 0.0)
```